pkg/catalog/catalog.go

```diff
@@ -52,51 +52,37 @@ func Search(client *occlient.Client, name string) ([]string, error) {
 	return result, nil
 }
 
-// Exists returns true if the given component type is valid, false if not
-func Exists(client *occlient.Client, componentType string) (bool, error) {
+// Exists returns true if the given component type and the version are valid, false if not
+func Exists(client *occlient.Client, componentType string, componentVersion string) (bool, bool, error) {
 
 	catalogList, err := List(client)
```

Comment:
`List` should not be called. odo should just try to get the builder image without listing them all.

Reply:
removed